juju/model.py

```diff
@@ -43,32 +45,88 @@ def get_relation(self, relation_name, relation_id=None):
                 # errors, ideally integrating the error catching with Juju's mechanisms.
                 raise TooManyRelatedApps(relation_name, num_related, 1)
 
+    def _is_local_unit(self, unit_name):
+        return unit_name == self._local_unit_name
```

Comment:
It's not worth having a function that looks like
```
        unit._is_local_unit(some_unit.name)
```
When its single job is replacing the line
```
        unit._local_unit_name == some_unit.name
```